tools/server/webui/package.json

```diff
@@ -1,66 +1,85 @@
-    "@tailwindcss/postcss": "^4.1.1",
-    "@tailwindcss/vite": "^4.1.1",
-    "@vscode/markdown-it-katex": "^1.1.1",
-    "autoprefixer": "^10.4.20",
-    "daisyui": "^5.0.12",
-    "dexie": "^4.0.11",
-    "highlight.js": "^11.10.0",
-    "katex": "^0.16.15",
-    "pdfjs-dist": "^5.2.133",
-    "postcss": "^8.4.49",
-    "react": "^18.3.1",
-    "react-dom": "^18.3.1",
-    "react-dropzone": "^14.3.8",
-    "react-hot-toast": "^2.5.2",
-    "react-markdown": "^9.0.3",
-    "react-router": "^7.1.5",
-    "rehype-highlight": "^7.0.2",
-    "rehype-katex": "^7.0.1",
-    "remark-breaks": "^4.0.0",
-    "remark-gfm": "^4.0.0",
-    "remark-math": "^6.0.0",
-    "tailwindcss": "^4.1.1",
-    "textlinestream": "^1.1.1",
-    "vite-plugin-singlefile": "^2.0.3"
-  },
-  "devDependencies": {
-    "@eslint/js": "^9.17.0",
-    "@types/markdown-it": "^14.1.2",
-    "@types/node": "^22.13.1",
-    "@types/react": "^18.3.18",
-    "@types/react-dom": "^18.3.5",
-    "@vitejs/plugin-react": "^4.3.4",
-    "eslint": "^9.17.0",
-    "eslint-plugin-react-hooks": "^5.0.0",
-    "eslint-plugin-react-refresh": "^0.4.16",
-    "fflate": "^0.8.2",
-    "globals": "^15.14.0",
-    "prettier": "^3.4.2",
-    "sass-embedded": "^1.83.4",
-    "typescript": "~5.6.2",
-    "typescript-eslint": "^8.18.2",
-    "vite": "^6.0.5"
-  },
-  "prettier": {
-    "trailingComma": "es5",
-    "tabWidth": 2,
-    "semi": true,
-    "singleQuote": true,
-    "bracketSameLine": false
-  }
+	"name": "webui",
+	"private": true,
+	"version": "0.0.1",
+	"type": "module",
+	"scripts": {
+		"dev": "vite dev --host 0.0.0.0 & storybook dev -p 6006 --ci",
+		"build": "vite build && ./scripts/post-build-cleanup.sh",
+		"preview": "vite preview",
+		"prepare": "svelte-kit sync || echo ''",
+		"check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
+		"check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
+		"format": "prettier --write .",
+		"lint": "prettier --check . && eslint .",
+		"test:unit": "vitest",
+		"test": "npm run test:unit -- --run && npm run test:e2e",
+		"test:e2e": "playwright test",
+		"storybook": "storybook dev -p 6006",
+		"build-storybook": "storybook build"
+	},
+	"devDependencies": {
+		"@chromatic-com/storybook": "^4.0.1",
+		"@eslint/compat": "^1.2.5",
+		"@eslint/js": "^9.18.0",
+		"@internationalized/date": "^3.8.2",
+		"@lucide/svelte": "^0.515.0",
+		"@playwright/test": "^1.49.1",
+		"@storybook/addon-a11y": "^9.0.17",
+		"@storybook/addon-docs": "^9.0.17",
+		"@storybook/addon-svelte-csf": "^5.0.7",
+		"@storybook/addon-vitest": "^9.0.17",
+		"@storybook/sveltekit": "^9.0.17",
+		"@sveltejs/adapter-static": "^3.0.8",
+		"@sveltejs/kit": "^2.22.0",
+		"@sveltejs/vite-plugin-svelte": "^6.0.0",
+		"@tailwindcss/forms": "^0.5.9",
+		"@tailwindcss/typography": "^0.5.15",
+		"@tailwindcss/vite": "^4.0.0",
+		"@types/node": "^22",
+		"@vitest/browser": "^3.2.3",
+		"bits-ui": "^2.8.11",
```

Comment:
Why have you replaced [DaisyUI](https://daisyui.com)?

Reply:
I chose to go with [ShadCN for Svelte](https://shadcn-svelte.com/) which uses Bits UI under the hood.